sample/OSCON17/event

```diff
@@ -0,0 +1,77 @@
+{
+	"id": 3,
+	"name": "OSCON open source convention",
+	"latitude": 30.263569,
+	"longitude": -97.739606,
+	"location_name": "Austin Convention Center, Austin, TX ",
+	"start_time": "2017-05-08T13:00:00-05:00",
+	"end_time": "2017-05-11T20:00:00-05:00",
+	"timezone": "Austin/Texas",
+	"description": "Open source is everywhere—from the bottom of the programming stack to the top. OSCON brings the software engineering and developer community together to explore what's new and useful in open source languages, tools, and techniques. It's the best place to sharpen your skills and discover important trends, making you better at what you do and igniting your love of code.",
+	"background_image": "https://raw.githubusercontent.com/mahikaw/Fossasia-sample-images/master/OSCON_2017_EverythingOpenSource.png",
+	"logo": "https://raw.githubusercontent.com/mahikaw/Fossasia-sample-images/master/log.png",
```

Comment:
I think you can use the image in folder instead of using the links, like you have used in other places.